YANG/ccamp/otn-tunnel/ietf-otn-tunnel.yang

```diff
@@ -493,11 +573,24 @@ module ietf-otn-tunnel {
     }
   }
 
+  /* Augment label-step of the label restrictions for the forwarding
+     direction of path-in-segment of primary reverse path */
+  augment "/te:te/te:tunnels/te:tunnel/"
+        + "te:primary-paths/te:primary-path/"
+        + "te:primary-reverse-path/"
+        + "te:path-in-segment/te:label-restrictions/"
+        + "te:label-restriction/te:label-step/te:technology" {
```

Comment:
Why no augmentation for label-start and label-end?

Reply:
Clarified that the label-start and label-end augmentations are already defined. This PR is adding also the augmentations for the label-step.